language/stdlib/modules/doc/AccountFreezing.md

```diff
@@ -619,6 +619,6 @@ Only (un)freeze functions can change the freezing bits of accounts [[H7]][PERMIS
 
 
 [//]: # ("File containing references which can be used from documentation")
-[ACCESS_CONTROL]: https://github.com/libra/lip/blob/master/lips/lip-2.md
-[ROLE]: https://github.com/libra/lip/blob/master/lips/lip-2.md#roles
-[PERMISSION]: https://github.com/libra/lip/blob/master/lips/lip-2.md#permissions
+[ACCESS_CONTROL]: https://github.com/diem/lip/blob/master/lips/lip-2.md
```

Comment:
lib --> dip, https://github.com/diem/dip

Reply:
heh, must have happened in the last few hours, sure.